pypeit/telescopes.py

```diff
@@ -78,6 +78,13 @@ def __init__(self):
                                                latitude=32.701308,
                                                elevation=3221.0)
 
+class MDMTelescopePar(TelescopePar):
+    def __init__(self):
+        super(MDMTelescopePar, self).__init__(name='MDM',
+                                              longitude=111.616111,
+                                              latitude=31.9516666,
+                                              elevation=2098.)                # Used KPNO not MDM
```

Comment:
This is a bit nit picky, but, if this is really for KPNO, we should try to call this out better.  Can we instead call this `KPNOTelescope` or something, and then add a comment in the MDM spectrograph that says we're using KPNO but should change it to MDM, and then add a GitHub issue?

Reply:
modified